tsconfig.json

```diff
@@ -12,7 +12,7 @@
         "outDir": "./lib",
         "declaration": true,
         "jsx": "react",
-        "lib": ["es2021", "dom", "dom.iterable"],
+        "lib": ["es2022", "dom", "dom.iterable"],
```

Comment:
Unintentional commit?

Reply:
Nope, it is necessary for the types for `Intl.Segmenter`